exonum/src/storage/values.rs

```diff
@@ -240,3 +241,59 @@ impl StorageValue for String {
         String::from_utf8(value.into_owned()).unwrap()
     }
 }
+
+/// Uses little-endian encoding.
+impl CryptoHash for SystemTime {
+    fn hash(&self) -> Hash {
+        let duration = self.duration_since(UNIX_EPOCH).unwrap();
+        let secs = duration.as_secs();
+        let nanos = duration.subsec_nanos();
+
+        let mut buffer = vec![0; 12];
```

Comment:
I am not sure that allocating a twelve element vector is a great idea :) Could we use `[0u8; 12]` here?